src/stores/api/api.store.tsx

```diff
@@ -8,22 +8,34 @@ export interface APIModel {
   classes: Record<string, any> | null;
   class: Record<string, any> | null;
   abilities: Record<string, any> | null;
+  races: Record<string, any> | null;
+  race: Record<string, any> | null;
+  allEquipment: Record<string, any> | null;
 
   generateDndClass: Action<APIModel, DnDClass>;
   generateClasses: Thunk<APIModel>;
   generateClass: Thunk<APIModel, string>;
   generateAbilities: Thunk<APIModel>;
+  generateRaces: Thunk<APIModel>;
+  generateRace: Thunk<APIModel, string>;
```

Comment:
Is there a list of `race` anywhere we can use as type hint instead of just `string`?

Reply:
generate races is a list of races. each one just happens to be an object
{
index: string,
name: string,
url: string
}

Reply:
Ah, then we search by string, got you